lib/galaxy/tools/parameters/basic.py

```diff
@@ -128,21 +128,21 @@ def get_dependencies( self ):
         """
         return []
 
-    def to_json( self, value, app ):
+    def to_json( self, value, app, security ):
         """Convert a value to a string representation suitable for persisting"""
         return unicodify( value )
 
     def to_python( self, value, app ):
         """Convert a value created with to_json back to an object representation"""
         return value
 
-    def value_to_basic( self, value, app ):
+    def value_to_basic( self, value, app, security=False ):
```

Comment:
I'd suggest security default to ``None`` instead of ``False`` - since this is conditionally passed an object.